src/renderers/shaders/ShaderLib/meshbasic.glsl.js

```diff
@@ -68,9 +68,8 @@ uniform float opacity;
 
 void main() {
 
-	#include <clipping_planes_fragment>
-
 	vec4 diffuseColor = vec4( diffuse, opacity );
+	#include <clipping_planes_fragment>
```

Comment:
I'm curious, what's the reason for moving the call to the clipping code below the calculation for the diffuse color, in many of the shaders?

Reply:
The clipping fragment code block now [multiples into the diffuse color alpha](https://github.com/mrdoob/three.js/pull/22172/files#diff-510604884d1277c044a3c6f6fc1994ae2baaedd6e7cc74a24df80017074dd52cR43) to perform the alpha test AA.

Reply:
Ah, makes sense. Thanks for explaning